src/surfing_penguin/db_interface.py

```diff
@@ -44,24 +44,29 @@ def load_user(id):
     return session.query(User).get(int(id))
 
 
-class Qstnr(object):
-    def __init__(self):
-        self.iter = 0
-        self.questions = []
+class SurveyFunctions(object):
+    def new_survey(name):
+        survey = Survey(name)
+        session.add(survey)
+        session.commit()
+        return survey
 
-    def get(self, id):
-        for qst in self.questions:
-            if qst['id'] == id:
-                return qst
-        api.abort(404, "question {} doesn't exist".format(id))
+    def get_all_surveys():
+        surveys = session.query(Survey).all()
+        return surveys
 
-    def new_qst(self, data):
-        question = data
-        self.iter = self.iter + 1
-        question['id'] = self.iter
-        self.questions.append(question)
-        return question
+    def get_all_questions(name):
+        s = session.query(Survey).filter_by(surveyname=name).first()
+        return s.questions
 
-    def delete(self, id):
-        question = self.get(id)
-        self.question.remove(question)
+    def new_question(survey, data):
+        # data should be a dictionary, with key "content" and "title"
```

Comment:
It would be better to explain the inputs in standard format if it's complicated. It could be used to generate document automatically by Sphinx (http://www.sphinx-doc.org/en/master/contents.html) as well.
For example:
```
"""
(Explain the function here)

Args:
      survey: Survey object in ORM
     data: a dictionary with key 'content' and 'title'
Returns:
    (explain if there is one)
"""
```
Also questions should include other attributes later. Please add a TODO for this.

Reply:
Okay, I saw the TODO in model.py